src/plugins/export/export.js

```diff
@@ -24,44 +25,79 @@ class Export extends Plugin {
+				}
 				const csv = new Csv();
 				const data = csv.write(this.rows, this.columns);
-				download(this.id, data, `text/${this.type}`);
+				download(fileSaver, this.id, data, `text/${this.type}`);
 			}
 		});
 		this.json = new Command({
 			canExecute: () => this.type === 'json',
 			execute: () => {
+				const fileSaver = this.imports.file;
+				if (!fileSaver) {
+					throw new AppError('fileSaver', 'To use export plugin for pdf format please add https://github.com/eligrey/FileSaver.js library to your project');
+				}
 				const json = new Json();
 				const data = json.write(this.rows, this.columns);
-				download(this.id, data, `text/${this.type}`);
+				download(fileSaver, this.id, data, `text/${this.type}`);
 			}
 		});
 		this.xml = new Command({
 			canExecute: () => this.type === 'xml',
 			execute: () => {
+				const fileSaver = this.imports.file;
+				if (!fileSaver) {
+					throw new AppError('fileSaver', 'To use export plugin for pdf format please add https://github.com/eligrey/FileSaver.js library to your project');
+				}
 				const xml = new Xml();
 				const data = xml.write(this.rows);
-				download(this.id, data, `application/${this.type}`);
+				download(fileSaver, this.id, data, `application/${this.type}`);
 			}
 		});
 		this.xlsx = new Command({
 			canExecute: () => this.type === 'xlsx',
 			execute: () => {
-				const xlsx = new Xlsx();
+				const lib = this.imports.xlsx;
+				const fileSaver = this.imports.file;
+
+				if (!lib) {
+					throw new AppError('xlsx', 'To use export plugin for xlsx format please add http://github.com/SheetJS/js-xlsx library to your project');
+				}
+				if (!fileSaver) {
+					throw new AppError('fileSaver', 'To use export plugin for pdf format please add https://github.com/eligrey/FileSaver.js library to your project');
+				}
+				const xlsx = new Xlsx(lib);
 				const data = xlsx.write(this.rows, this.columns);
-				download(this.id, data, 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet', 'xlsx');
+				download(fileSaver, this.id, data, 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet', 'xlsx');
 			}
 		});
 		this.pdf = new Command({
 			canExecute: () => this.type === 'pdf',
 			execute: () => {
-				const pdf = new Pdf();
+				const fileSaver = this.imports.file;
+				const pdfLib = this.imports.pdf;
+				const autotableLib = this.imports.pdf;
+
+				if (!fileSaver) {
+					throw new AppError('fileSaver', 'To use export plugin for pdf format please add https://github.com/eligrey/FileSaver.js library to your project');
+				}
+				if (!pdfLib || !autotableLib) {
+					throw new AppError('jsPDF', 'To use export plugin for pdf format please add https://github.com/MrRio/jsPDF and https://github.com/simonbengtsson/jsPDF-AutoTable libraries to your project');
+				}
+				const pdf = new Pdf(pdfLib);
 				pdf.write(this.rows, this.columns, this.id);
 			}
 		});
 	}
 
+	get imports() {
```

Comment:
better approach:
importLib(name){
if (not found) throw error

return lib;
}